ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,296 @@
+        }
+        .environment(\.defaultMinListRowHeight, 0)
+        .frame(width: .infinity)
+    }
+}
+
+extension MSFListView {
+    /// View for List Cells
+    struct MSFListCellView: View {
+        var cell: MSFListVnextCellData
+        var tokens: MSFListTokens
+        @State var isTapped: Bool = false
+
+        init(cell: MSFListVnextCellData, tokens: MSFListTokens) {
+            self.cell = cell
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            HStack(spacing: 0) {
+                if let leadingIcon = cell.leadingIcon {
+                    Image(uiImage: leadingIcon)
+                        .resizable()
+                        .frame(width: tokens.iconSize, height: tokens.iconSize)
+                        .padding(.trailing, tokens.iconInterspace)
+                }
+                VStack(alignment: .leading) {
+                    if let title = cell.title {
+                        Text(title)
+                            .font(Font(tokens.textFont))
+                            .foregroundColor(Color(tokens.leadingTextColor))
+                            .lineLimit(cell.titleLineLimit)
+                    }
+                    if let subtitle = cell.subtitle {
+                        if subtitle != "" {
+                            Text(subtitle)
+                                .font(Font(tokens.subtitleFont))
+                                .foregroundColor(Color(tokens.subtitleColor))
+                                .lineLimit(cell.titleLineLimit)
+                        }
+                    }
+                }
+                Spacer()
+                HStack {
+                    if let trailingIcon = cell.trailingIcon {
+                        if trailingIcon != .none {
+                            let isDisclosure = trailingIcon == .disclosure
+                            let disclosureSize = tokens.disclosureSize
+                            let iconSize = tokens.iconSize
+                            Image(uiImage: trailingIcon.icon!)
+                                .resizable()
+                                .foregroundColor(Color(isDisclosure ? tokens.disclosureIconForegroundColor : tokens.trailingItemForegroundColor))
+                                .frame(width: isDisclosure ? disclosureSize : iconSize,
+                                       height: isDisclosure ? disclosureSize : iconSize)
+                                .padding(.leading, isDisclosure ? tokens.disclosureInterspace : tokens.iconInterspace)
+                        }
+                    }
+                }
+            }
+            .contentShape(Rectangle())
+            .padding(.leading, tokens.horizontalCellPadding)
+            .padding(.trailing, tokens.horizontalCellPadding)
+            .frame(minHeight: tokens.layoutType.height)
+            .listRowInsets(EdgeInsets())
+            .background(isTapped ? Color(tokens.highlightedBackgroundColor) : Color.clear)
```

Comment:
The only thing I still could not figure out is how to customize the background color for when the button is pressed.